cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -1473,33 +1472,6 @@ def test_item_factory(self):
         item = ItemFactory.create(parent_location=course.location)
         self.assertIsInstance(item, SequenceDescriptor)
 
-    def test_course_index_view_with_course(self):
-        """Test viewing the index page with an existing course"""
-        CourseFactory.create(display_name='Robot Super Educational Course')
```

Comment:
Tests creating a new course and then verifying that it shows in the course listing on the studio home page.
https://github.com/edx/edx-platform/blob/master/common/test/acceptance/tests/studio/test_studio_course_create.py#L97

Could add here clicking on the listing to make sure that it opens the course.